examples/k8s-2-configurable/tox.ini

```
@@ -0,0 +1,66 @@
+# Copyright 2025 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[tox]
+no_package = True
+skip_missing_interpreters = True
+min_version = 4.0.0
+env_list = unit
+
+[vars]
+src_path = {tox_root}/src
+tests_path = {tox_root}/tests
+
+[testenv]
+set_env =
+    PYTHONPATH = {tox_root}/lib:{[vars]src_path}
+    PYTHONBREAKPOINT=pdb.set_trace
+    PY_COLORS=1
+pass_env =
+    PYTHONPATH
+    CHARM_BUILD_DIR
+    MODEL_SETTINGS
+
+[testenv:unit]
+description = Run unit tests
+deps =
+    pytest
+    cosl
+    coverage[toml]
+    ops[testing]
+    -r {tox_root}/requirements.txt
+commands =
+    coverage run --source={[vars]src_path} \
+                 -m pytest \
+                 --tb native \
+                 -v \
+                 -s \
+                 {posargs} \
+                 {[vars]tests_path}/unit
```

Feel free to leave this as is, since I guess we might update these files anyway to use `uv`, but I really prefer formatting like this:
```suggestion
    coverage run --source={[vars]src_path} -m pytest \
        -v \
        -s \
        --tb native \
        {posargs} \
        {[vars]tests_path}/unit
```
That is, the first line is our `pytest` invocation (via `coverage`), and the `pytest` args are just indented 4 spaces from that. The argument lines can then be identical for the `integration` environment's `pytest` command (+/- `--log-cli-level=INFO`, and with the test directory differing).

---

Nice tip, thanks! I've just done this.

Are you also suggesting that I change the `integration` environments?

```
commands =
    pytest \
        -v \
        -s \
        --tb native \
        --log-cli-level=INFO \
        {posargs} \
        {[vars]tests_path}/integration
```

---

Done!